src/_icons.scss

```diff
@@ -41,7 +41,7 @@ $iconMap: (
 // Square large
 .ico_squareLarge {
   fill: currentColor;
-  height: $auro-size-lg;
+  height: var(--auro-size-lg, var(--ds-size-300, 1.5rem))
```

Comment:
I think these also should all be --ds not --auro. Looks like just this one got missed.

Reply:
No, that's intentional. That's part of the seamless fallback strategy. There is code out there that only knows to look for the `--auro` token. So putting `--auro` first keeps things working as expected, but allows people to switch over to `--ds` and things will still work.

Reply:
Also, there is no `--ds-size-lg`, that was removed with the new release.

Reply:
Got it, I see so the fallback for --auro is first the -ds-size-300 then the SASS var.